src/io_reading_files/Customer.java

```diff
@@ -0,0 +1,25 @@
+package io_reading_files;
```

Comment:
we do not name packages like this. no snake case. should be domain name like: de.company.component, etc.